### PR TITLE
Enhance the load completion.

### PR DESCRIPTION
**Pull request by Green-m**
The completion for `load` would list all plugins no matter whether it loaded or not.

Add a new parameter for `load` to show all loaded plugins.

### CODE REVIEW

File: lib/msf/ui/console/command_dispatcher/core.rb [hunk 1]
```diff
@@ -697,6 +697,7 @@ def cmd_load_help
     print_line
     print_line "Loads a plugin from the supplied path."
     print_line "For a list of built-in plugins, do: load -l"
+    print_line "For a list of loaded pligins, do: load -s"
```
Comment (bcoles) [inline lib/msf/ui/console/command_dispatcher/core.rb]:
```suggestion
    print_line "For a list of loaded plugins, do: load -s"
```

File: lib/msf/ui/console/command_dispatcher/core.rb [hunk 2]
```diff
@@ -800,8 +803,8 @@ def cmd_load_tabs(str, words)
     else
       tabs += tab_complete_filenames(str,words)
     end
-    return tabs.map{|e| e.sub(/.rb/, '')}
 
+    return tabs.map{|e| e.sub(/.rb/, '')} - framework.plugins.map(&:name)
```
[the remark this answers is not in the document]
Comment (Green-m) [reply lib/msf/ui/console/command_dispatcher/core.rb]: Good catch!

**Comment by busterb** [comment]
```
msf5 exploit(multi/handler) > load 
load aggregator        load ips_filter        load openvas           load sounds
load alias             load komand            load pcap_log          load sqlmap
load auto_add_route    load lab               load request           load thread
load beholder          load libnotify         load rssfeed           load token_adduser
load db_credcollect    load msfd              load sample            load token_hunter
load db_tracker        load msgrpc            load session_notifier  load wiki
load event_tester      load nessus            load session_tagger    load wmap
load ffautoregen       load nexpose           load socket_logger     
msf5 exploit(multi/handler) > load beholder 
[*] Successfully loaded plugin: beholder
msf5 exploit(multi/handler) > load alias 
[*] Successfully loaded plugin: alias
msf5 exploit(multi/handler) > load 
load aggregator        load lab               load request           load thread
load auto_add_route    load libnotify         load rssfeed           load token_adduser
load db_credcollect    load msfd              load sample            load token_hunter
load db_tracker        load msgrpc            load session_notifier  load wiki
load event_tester      load nessus            load session_tagger    load wmap
load ffautoregen       load nexpose           load socket_logger     
load ips_filter        load openvas           load sounds            
load komand            load pcap_log          load sqlmap            
msf5 exploit(multi/handler) > load aggregator 
[-] Failed to load plugin from /home/bcook/projects/metasploit-framework/plugins/aggregator: cannot load such file -- metasploit/aggregator
msf5 exploit(multi/handler) > load db_tracker 
[*] Successfully loaded plugin: db_tracker
msf5 exploit(multi/handler) > load 
load aggregator        load lab               load pcap_log          load sounds
load auto_add_route    load libnotify         load request           load sqlmap
load db_credcollect    load msfd              load rssfeed           load thread
load event_tester      load msgrpc            load sample            load token_adduser
load ffautoregen       load nessus            load session_notifier  load token_hunter
load ips_filter        load nexpose           load session_tagger    load wiki
load komand            load openvas           load socket_logger     load wmap
```

**Comment by busterb** [comment]
## Release Notes

This improves the msfconsole `load` command to tab-complete with a list of unloaded plugins, as well as adding a `-s` switch for displaying loaded plugins.

**Pull request merged**